packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.13+6
+
+* Fixes incorrect use of `NSError` that could cause crashes on launch.
```

Comment:
Nit: ... cause crashes on launch *on iOS 17*

Reply:
I unfortunately don't know if it's "on iOS 17" or "when compiled against the iOS 17 SDK" or some other slight variant of that, which is why I left it vague.